src/mono/cmake/configure.cmake

```diff
@@ -99,9 +99,9 @@ endif()
 
 check_include_files("sys/types.h;sys/user.h" HAVE_SYS_USER_H)
 
-if(NOT HOST_DARWIN)
-  # getentropy was introduced in macOS 10.12 / iOS 10.0
-  ac_check_funcs (getentropy)
+if(HOST_IOS OR HOST_MACCAT)
+  # getentropy isn't allowed in the AppStore: https://github.com/rust-lang/rust/issues/102643
+  set(HAVE_GETENTROPY 0)
```

Comment:
If it's not allowed on the AppStore then I think we should keep excluding it for macOS/tvOS too (even if macOS is not supported config for Mono anymore).

Reply:
Yeah one thing that might be confusing is that HOST_IOS includes tvOS in the mono cmake build system. Longer term I'd like to unify differences like this with the rest of the repo.